.github/workflows/tag.yml

```diff
@@ -0,0 +1,24 @@
+name: RELEASE
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
can we be a bit more expressive? it should really be `v\d+*` or something like that.